pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1671,6 +1671,12 @@ type AlertmanagerEndpoints struct {
 	//
 	// +optional
 	EnableHttp2 *bool `json:"enableHttp2,omitempty"`
+
+	// Relabeling configs for alerts before sending to the Alertmanager
+	//
+	// It requires Prometheus >= v2.51.0.
+	// +optional
+	AlertRelabelConfigs []RelabelConfig `json:"alertRelabelConfigs,omitempty"`
```

Comment:
to be consistent with the existing relabeling fields in ServiceMonitor and PodMonitor
```suggestion
	AlertRelabelConfigs []RelabelConfig `json:"alertRelabelings,omitempty"`
```